source/docs/articles/local/drupal-drush-command-line-utility.md

```diff
@@ -56,15 +37,15 @@ To get started, go to your Pantheon Dashboard and click **Download all Drush ali
 
 ###Install the Pantheon Drush Aliases
 
-If you are on Linux/MacOS environment, put the generated pantheon.aliases.drushrc.php in either the `.drush` directory in your home, or the `aliases` directory of your local Drush installation.
+If you are on Linux/Mac OS environment, put the generated pantheon.aliases.drushrc.php in either the `.drush` directory in your home, or the `aliases` directory of your local Drush installation. 

```

Comment:
Don't tell people to put their custom files inside Drush! There is an issue in the Drush queue to remove this search location.  The aliases should always go in $HOME/.drush.  (Or /etc/drush, if you want to get into global locations; maybe best to skip that, though.)